addons/breathing/functions/fnc_handleBreathing.sqf

```diff
@@ -202,3 +202,21 @@ if (!local _unit) then {
         };
     };
 }, 3, [_unit]] call CBA_fnc_addPerFrameHandler;
+
+
+
+[{
+    params ["_args", "_idPFH"];
+    _args params ["_unit"];
+
+    private _status = _unit getVariable [QGVAR(airwayStatus), 100];
+
+    if(!(alive _unit) || _status > 100 || IS_UNCONSCIOUS(_unit)) exitwith {
```

Comment:
shouldnt it be "status >= 100"? i dont see a reason to keep that pfh running when someone SpO2 status is 100
and add space after if

Reply:
Thanks for the hint completly oversaw that.

Reply:
But SpO2 can not be above 100, correct?